config/webpack-config.js

```diff
@@ -66,8 +66,19 @@ var onDemandConf = {
   ]
 };
 
+var nonResponsiveConf = {
+  context: path.join( __dirname, '/../', paths.unprocessed,
+                      JS_ROUTES_PATH + '/on-demand' ),
+  entry:   './header.js',
+  output: {
+    path:     path.join( __dirname, 'js' ),
+    filename: '[name]'
+  }
+};
+
 module.exports = {
-  onDemandConf: onDemandConf,
-  ieConf:       ieConf,
-  modernConf:   modernConf
+  nonResponsiveConf: nonResponsiveConf,
```

Comment:
I'm leaning toward naming this something like `onDemandUnminifiedConf` so it describes what the config does, not where it's used. There's nothing about the configuration that makes it non-responsive. It's the on-demand scripts, just without minification. Its use in https://github.com/cfpb/cfgov-refresh/pull/2170/files#diff-9d68bb3ecce3a165761fdcb2b7966095R94 is what makes it non-responsive, and that function has a name that conveys that. Haha, naming is the hardest ;)